code/game/machinery/deployable.dm

```diff
@@ -79,6 +85,15 @@
 				return
 	return ..()
 
+/obj/structure/barricade/wooden/crowbar_act(mob/living/user, obj/item/tool)
+	balloon_alert(user, "deconstructing barricade...")
+	if(!tool.use_tool(src, user, 2 SECONDS, volume=100))
```

Comment:
default volume is 50, is this loudness necessary?

Reply:
Applied with https://github.com/tgstation/tgstation/pull/69676/commits/5a7118899455875cbe0529e69f9117aa6055e92a